app_dart/lib/src/request_handlers/check_for_waiting_pull_requests.dart

```diff
@@ -257,12 +257,14 @@ bool _checkStatuses(
     }
   }
   for (Map<String, dynamic> check in checks) {
-    final String name = check['app']['name'];
-    if (check['conclusion'] != 'SUCCESS') {
-      allSuccess = false;
-      if (check['conclusion'] == 'FAILURE' &&
-          !notInAuthorsControl.contains(name)) {
-        failures.add(name);
+    for (Map<String, dynamic> checkRun in check['checkRuns']['nodes']) {
+      if (checkRun['conclusion'] != 'SUCCESS') {
+        final String name = checkRun['name'];
+        allSuccess = false;
+        if (checkRun['conclusion'] == 'FAILURE' &&
+            !notInAuthorsControl.contains(name)) {
+          failures.add(name);
```

Comment:
BTW how do you check that CI is in progress? Something like `!allSuccess && failures.isEmpty`?

Reply:
We delay actually checking the PR for an hour after the last push event. 

The idea is that should be more than enough time to get CI started, and avoids needing to hard code anything about how many jobs we expect to run.